package-lock.json

```diff
@@ -316,6 +315,9 @@
 		"@automattic/material-design-icons": {
 			"version": "file:packages/material-design-icons"
 		},
+		"@automattic/media-library": {
```

Comment:
I wonder if we should completely remove this package, as this project was dropped. @griffbrad what do you think?

Reply:
👍🏻 Agreed.  We should drop it for now.